### DISCUSSION TITLE
Prisma Unit testing in javascript language

### DISCUSSION THREAD

**Discussion by anoirsa** [prisma/prisma#10875] [Q&A]
Hello 
I'm doing a project in nodejs and using prisma io in it 
but I have got a problem I want to make test files (Unit testing) to test create method for example
I have seen docs in prisma io website but they are in typescript not in javascript

does anyone have any references of prisma io unit testing in javascript ?

Thank you very much

**Comment by colinpcurtis** [comment]
I found this link in the Prisma discussion forum https://github.com/prisma/prisma/discussions/2083.

There are also examples in the [prisma-examples](https://github.com/prisma/prisma-examples/tree/latest/typescript/testing-express) repo that give a way to do testing.

**Comment by Octobug** [comment]
I've made a `Nodejs` demo which is equivalent to the official guide - [Unit testing (`TypeScript` version)](https://www.prisma.io/docs/guides/testing/unit-testing).

See: https://github.com/OctobugDemo/nodejs-prisma-unit-test

It might help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
